lib/uploadcare/entity/document_converter.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Uploadcare
+  module Entity
+    # This serializer lets a user convert uploaded documents
+    # @see https://uploadcare.com/api-refs/rest-api/v0.6.0/#operation/documentConvert
+    class DocumentConverter < Entity
+      client_service Conversion::DocumentConversionClient
+      # Converts documents
+      #
+      # @param doc_params [Array] of hashes with params or [Hash]
+      # @option options [Boolean] :store (false) whether to store file on servers.
+      def self.convert(doc_params, **options)
+        params = doc_params.is_a?(Hash) ? [doc_params] : doc_params
+        Conversion::DocumentConversionClient.new.convert_many(params, **options)
+      end
+
+      # Returns a status of document conversion job
+      #
+      # @param token [Integer, String] token obtained from a server in convert method
+      def self.status(token)
+        Conversion::DocumentConversionClient.new.get_conversion_status(token)
```

Comment:
It looks like `client_service Conversion::DocumentConversionClient` is being used so that we can just write
```ruby
get_conversion_status(token)
```

Reply:
This changes breaks the logic. Somehow the result of the method becomes an empty hash:

**with `Conversion::DocumentConversionClient.new`:**
```ruby
def self.status(token)
  Conversion::DocumentConversionClient.new.get_conversion_status(token)
end
```

```console
Uploadcare::DocumentConverter.status(21705424)
# => Success({:status=>"finished", :error=>nil, :result=>{:uuid=>"aefdfb41-99af-42e0-a987-52bba1967b3b"}})
```

**without `Conversion::DocumentConversionClient.new`:**
```ruby
def self.status(token)
  get_conversion_status(token)
end
```

```console
Uploadcare::DocumentConverter.status(21705424)
# => {}
```

Reply:
OK.
What about
```ruby
def client
  Conversion::DocumentConversionClient.new
end
```

Reply:
:+1:  I added something like this [here](https://github.com/uploadcare/uploadcare-ruby/pull/76/files#diff-47ff036716a9083c7e66182a7c64b5808792cbd9d7a3e0ac4e9081eab160d8e7R28)